src/Sarif.Viewer.VisualStudio/CodeFlowToTreeConverter.cs

```diff
@@ -30,15 +30,17 @@ private static List<CallTreeNode> GetChildren(CodeFlow codeFlow, ref int current
                         children.Add(new CallTreeNode
                         {
                             Location = codeFlow.Locations[currentCodeFlowIndex],
-                            Children = GetChildren(codeFlow, ref currentCodeFlowIndex)
+                            Parent = parent
                         });
+                        children[children.Count - 1].Children = GetChildren(codeFlow, ref currentCodeFlowIndex, children[children.Count - 1]);
```

Comment:
I had to stare at this for a while to understand it: You need to pass the newly created node to _its own children_ as their parent, and the newly created node doesn't have a name yet, so you identify it as "the most recently added child". Let me think if this can be written more clearly...